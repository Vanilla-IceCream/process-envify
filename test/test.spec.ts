import { describe, it, expect } from 'vitest';

import envify from '../src';

describe('process-envify', () => {
  it('should inject strings', () => {
    const data = {
      BOOK_NAME: 'ECMAScript: Up and Running',
    };

    expect(envify(data)).toMatchSnapshot();
  });
});
