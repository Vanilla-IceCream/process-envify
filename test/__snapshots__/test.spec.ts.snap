// Vitest Snapshot v1

exports[`process-envify > should inject strings 1`] = `
{
  "process.env.BOOK_NAME": "\\"ECMAScript: Up and Running\\"",
}
`;
